web-components/package.json

```diff
@@ -8,8 +8,8 @@
     "copy-assets": "cp ./src/theme/fonts.css ./lib/theme/fonts.css && cp -R ./src/theme/assets ./lib/theme/",
     "watch": "yarn remove-previous-files && tsc --watch && yarn copy-assets",
     "build": "yarn remove-previous-files && tsc && yarn copy-assets",
-    "lint": "tsc --noEmit && eslint ./src  -c .eslintrc.js --max-warnings 0 --ext .ts,.tsx",
```

Comment:
This was indeed not necessary as it's possible to run eslint even if there're errors in tsc compilation.
That being said should we have this command elsewhere?
Like `"ts-check": "tsc --noEmit"`

Reply:
my thought was that any changes will be done through running the dev servers locally which should catch any compilation errors. We can certainly add a script to just type check, but i’m curious what you’d use it for?